src/dtoValidation/processors/LengthProcessor.java

```diff
@@ -0,0 +1,28 @@
+package dtoValidation.processors;
+
+import dtoValidation.annotations.Length;
+import dtoValidation.dto.Dto;
+import dtoValidation.error.Error;
+
+import java.lang.reflect.Field;
+
+public class LengthProcessor<T> implements AnnotationProcessor<T>{
+
+    @Override
+    public Error validate(T t) throws IllegalAccessException {
+        Field[] fields = t.getClass().getDeclaredFields();
+        for (Field field : fields) {
+            if (field.isAnnotationPresent(Length.class)) {
+                field.setAccessible(true);
+                String fieldName = field.get(t).toString();
+                if (fieldName.length() < field.getAnnotation(Length.class).min()) {
+                    return Error.MinLengthError;
```

Comment:
Instead of returning enum value, just return a list of errors. In this case, it is possible to have a multiple errors